front/lib/api/assistant/feedback.ts

```diff
@@ -26,164 +23,118 @@ export type AgentMessageFeedbackType = {
   userId: number;
   thumbDirection: AgentMessageFeedbackDirection;
   content: string | null;
+  createdAt: Date;
+  agentConfigurationId: string;
+  agentConfigurationVersion: number;
+  isConversationShared: boolean;
+};
+
+export type AgentMessageFeedbackWithMetadataType = AgentMessageFeedbackType & {
+  conversationId: string | null;
+  userName: string;
+  userEmail: string;
+  userImageUrl: string | null;
 };
 
 export async function getConversationFeedbacksForUser(
   auth: Authenticator,
   conversation: ConversationType | ConversationWithoutContentType
-): Promise<Result<AgentMessageFeedbackType[], ConversationError>> {
+): Promise<Result<AgentMessageFeedbackType[], ConversationError | Error>> {
   const owner = auth.workspace();
   if (!owner) {
-    throw new Error("Unexpected `auth` without `workspace`.");
+    return new Err(new Error("workspace_not_found"));
```

Comment:
We can throw here. and even more compact is to use:

auth.getNonNullableWorskspace()